pyanaconda/modules/common/structures/language.py

```diff
@@ -0,0 +1,67 @@
+#
+# DBus structures for the language and locale data.
+#
+# Copyright (C) 2022  Red Hat, Inc.  All rights reserved.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received the copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+from dasbus.structure import DBusData
+from dasbus.typing import *  # pylint: disable=wildcard-import
+
+__all__ = ["LanguageData"]
+
+
+class LanguageData(DBusData):
+    """Language data."""
+
+    def __init__(self):
+        self._english = ""
+        self._lid = ""
+        self._native = ""
+
+    @property
+    def english(self) -> Str:
+        """English name for the given language/locale
+
+        :return: the english name of the given language/locale
+        """
+        return self._english
+
+    @english.setter
+    def english(self, english: Str):
+        self._english = english
+
+    @property
+    def lid(self) -> Str:
+        """Language identifier
+
+        :return: the language/locale identifier
```

Comment:
This structure is used for Languages and for Locales, meaning that the values are different according to the usecase.

It will be `en` when used for the AvailableTranslations list, and it will be `en_US.UTF-8` if it will be used for the struct in the GetLanguageLocales call.